molecule/src/molecule_ionize.cpp

```diff
@@ -444,433 +419,423 @@ void MoleculePkaModel::_loadSimplePkaModel()
-        {PKA_BASIC, "7300021310000|6400020430000|640002042000064000204200007300021310000", 10.61f, 0.03f},
-        {PKA_BASIC, "7300021310000|6400020440000|6400020410000640002041000064000204200007300021310000", 10.85f, 0.00f},
-        {PKA_BASIC, "7300021330000|640002041000064000204100006400020420000|73000213300006400020420000", 9.27f, 1.02f},
-        {PKA_BASIC, "7300021330000|640002041000064000204200006400020420000|730002133000064000204100006400020410000", 10.35f, 0.00f},
-        {PKA_BASIC, "7300021311000|64000204110006400020421000|7300021311000730002131100064000204210007300021311000", 4.05f, 0.00f},
-        {PKA_BASIC, "7300021310000|6400020421000|640002041100064000204110007300021310000", 3.74f, 2.72f},
-        {PKA_BASIC, "7300021310000|6400020421000|640002041100064000204210007300021310000", 3.23f, 3.48f},
-        {PKA_BASIC, "7300021311000|64000204110006400020421000|6400020411000730002131100064000204110006400020411000", 12.68f, 0.00f},
-        {PKA_BASIC, "7300021311000|64000204210007300021320000|6400020411000640002042100073000213110007300021311000", 1.60f, 0.00f},
-        {PKA_BASIC, "7300021311000|64000204110006400020421000|6400020411000730002131100064000204210007300021320000", 2.27f, 0.00f},
-        {PKA_BASIC, "7300021320000|64000204110006400020421000|6400020411000730002132000064000204110006400020411000", 3.59f, 0.00f},
-        {PKA_BASIC, "7300021311000|64000204110006400020421000|6400020411000730002131100064000204110008200022220000", 3.28f, 0.00f},
-        {PKA_BASIC, "7300021310000|7300021320000|64000204210007300021310000", 8.79f, 0.00f},
-        {PKA_BASIC, "7300021330000|640002042000064000204200006400020420000|6400020421000730002133000064000204210006400020421000", 10.70f, 0.00f},
-        {PKA_BASIC, "7300021311000|64000204210006400020421000|64000204100006400020411000730002131100073000213100007300021311000", 4.82f, 0.00f},
-        {PKA_BASIC, "7300021330000|640002041000064000204200006400020430000|7300021330000640002042000064000204100006400020420000", 10.21f, 0.01f},
-        {PKA_BASIC, "7300021330000|640002042000064000204200006400020420000|6400020410000730002133000064000204200006400020420000", 9.06f, 1.39f},
-        {PKA_BASIC, "7300021330000|640002042000064000204200006400020420000|6400020420000730002133000064000204200006400020421000", 8.35f, 0.00f},
-        {PKA_BASIC, "7300021320000|64000204300006400020430000|64000204100006400020410000730002132000064000204100006400020410000", 11.05f, 0.00f},
-        {PKA_BASIC, "7300021330000|640002042000064000204200006400020420000|6400020410000730002133000064000204100006400020410000", 10.75f, 0.00f},
-        {PKA_BASIC, "7300021330000|640002042000064000204200006400020420000|6400020420000730002133000064000204200006400020420000", 8.50f, 1.97f},
-        {PKA_BASIC, "7300021320000|1440002044000014400020440000|6400020410000640002041000064000204100007300021320000640002041000064000204100006400020410000",
+    // These are basics
+    PkaDef advanced_pka_model_basic[] = {
```

Comment:
Let it be `static const` or even `static constexpr`. (we've passed the point of no return to C++11)

Reply:
It is static data. And therefore it const anyway.